guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -81,6 +82,12 @@ endif::[]
 --organization "_My_Organization_"
 ----
 +
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer activation-key product-content \
+--name "_My_Activation_Key_" \
+--organization "_My_Organization_"
+----
```

Comment:
This doesn't seem to be needed. Doesn't it just repeat the command directly above it?

edit: Oh, I see the one above has `--content-access-mode-all true`. I think we do need that for SCA, so this change should be removed and we should just keep the existing one.